VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -548,6 +548,9 @@ export class VideoCallService {
         this.pexipAPI.setParticipantText(uuid, text);
     }
 
+    transformLayout(layout: string) {
```

Comment:
could we add a test for this (I know it won't do much)? Maybe similar to what we have for `setParticipantOverlayText`